heudiconv/dicoms.py

```diff
@@ -55,6 +55,8 @@ def group_dicoms_into_seqinfos(files, file_filter, dcmfilter, grouping):
         nfl_after = len(files)
         lgr.info('Filtering out {0} dicoms based on their filename'.format(
             nfl_before-nfl_after))
+    import nibabel.nicom.csareader
+    nibabel.nicom.csareader.MAX_CSA_ITEMS = 400
```

Comment:
Ok with me, but may be we should look into either changing it in `nibabel` itself? 
if we are to patch it here, then we should make it conditional like
```python
nibabel.nicom.csareader.MAX_CSA_ITEMS = max(nibabel.nicom.csareader.MAX_CSA_ITEMS, 400)
```
so we do not bring it down if it is already set higher!  but again -- better be changed etc in nibabel since what if they just drop/rename that variable? then our change would either be of no effect